### PR TITLE
fix: AUT URL in runner header links to application URL

### LINKED ISSUE #21823: AUT header - URL should be a link
In 9.x this functions as a link:

Screen Shot 2022-04-11 at 1 58 34 PM

It’s set up as a form:

Screen Shot 2022-04-11 at 1 59 06 PM

In 10.0 we can probably just make this a genuine link with target `blank` since in 9.x it opens in new tab within the Cypress-owned browser instance. In 9.x the form field can be focused with the keyboard, but there is no way to trigger the link.

We should follow 9.x styles and underline this on hover using something like `indigo-500` for the color.

┆Issue is synchronized with this [Jira Bug](https://cypress-io.atlassian.net/browse/UNIFY-1556) by [Unito](https://www.unito.io)
┆Attachments: <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/11740">Screen Shot 2022-04-11 at 1.58.34 PM.png</a> | <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/11741">Screen Shot 2022-04-11 at 1.59.06 PM.png</a>
┆author: Mark Noonan
┆epic: 9.x Parity
┆friendlyId: UNIFY-1556
┆priority: Low
┆sprint: Backlog
┆taskType: Bug

### PR DESCRIPTION
**Pull request by astone123**
- Closes #21823

### User facing changelog

Runner header now links to application URL while running e2e tests

### Additional details

Before v10, the application URL in the AUT header was a link that the user could click which would open the current application URL in a new tab. That functionality was lost during updates to the header for v10.

### Steps to test

1. Run an e2e test
2. In the header "address bar", click the application URL
3. Verify that a new tab is opened at the application URL that was open in the runner during your test

### How has the user experience changed?

**Before**
Screen Shot 2022-06-07 at 3 38 06 PM

**After**
Screen Shot 2022-06-07 at 3 32 00 PM

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by lmiller1990 — APPROVED** [review]
Tried it out, works great :+1: let's ship it.

File: packages/app/src/runner/SpecRunnerHeaderOpenMode.vue [hunk 1]
```diff
@@ -23,9 +23,13 @@
         >
           <i-cy-crosshairs_x16 :class="[selectorPlaygroundStore.show ? 'icon-dark-indigo-500' : 'icon-dark-gray-500']" />
         </Button>
-        <div class="mr-12px max-w-100% grid text-gray-600 items-center truncate">
+        <a
+          target="_blank"
+          :href="autStore.url"
+          class="mr-12px max-w-100% grid text-indigo-500 hocus-link-default items-center truncate"
```
[the remark this answers is not in the document]
Comment (astone123) [reply packages/app/src/runner/SpecRunnerHeaderOpenMode.vue]: Ah good catch! I made the update

**Review by tbiethman — APPROVED** [review]

**Comment by lmiller1990** [comment]
If you merge in develop, windows CI should start working again.

**Comment by lmiller1990** [comment]
Going to merge this - windows fails, but those are fixed in https://github.com/cypress-io/cypress/pull/22234#pullrequestreview-1002252834, I don't think you've introduced any here.

**Pull request merged**